core/src/main/scala/org/bitcoins/core/protocol/ln/util/LnUtil.scala

```diff
@@ -16,19 +17,19 @@ trait LnUtil {
     * [[https://github.com/lightningnetwork/lightning-rfc/blob/master/11-payment-encoding.md#examples Bolt11]]
     * for examples.
     */
-  def createDataLength(bech32String: String): Vector[UInt5] = {
+  def createDataLength(bech32String: String): List[UInt5] = {
```

Comment:
Most changes in this file are because `List` has way better performance for construction and destruction of collections one by one. 

http://www.lihaoyi.com/post/BenchmarkingScalaCollections.html#lists-vs-vectors

```
Constructing a List item-by-item is 5-15x (!) faster than constructing a Vector item-by-item.

De-constructing a List item-by-item using .tail is 50-60x (!) faster than de-constructing a Vector item-by-item using .tail
```